core/src/main/scala/scala/js/exp/TraitsExp.scala

```diff
@@ -35,7 +35,7 @@ trait TraitsExp extends Traits with ProxyExp {
     }
 
     val implClazz = Class.forName(traitClazz.getName + "$class")
-    val parents = traitClazz.getInterfaces.filter(_ != implicitly[Manifest[scala.ScalaObject]].erasure)
+    val parents = traitClazz.getInterfaces.filter(_ != implicitly[Manifest[AnyRef]].runtimeClass)
```

Comment:
oh, that one. Because [ScalaObject](http://www.scala-lang.org/api/2.10.4/index.html#scala.ScalaObject) has been removed, and the closest thing to it is AnyRef.